docs/source/design/composite-time-series.rst

```diff
@@ -0,0 +1,386 @@
+    If someone came to you and said what was the level on XX Date/time what value would you give them?
+    That is the period of record. but for all dates and times.  
+
+
+
+Axioms
+------
+
+#. Composite Time Series are Irregular
+#. The definition of the composite time series is stored within the CWMS database
+#. The members of a composite time series define a continuous range
+   #. The date ranges of a member *MUST* not overlap
+   #. Each member *MUST* have a start date
+   #. The last member *MAY* have an end date indicating no more data will be available for this location and measure.
+   #. Data may have gaps, an explanation range *SHOULD* be provided. For data with regular gaps, e.g. season gauges
+      a description should be provided in the notes. A Link to a Location Level can be provided if the timing is well known.
+#. The members of a composite time measure the same thing. (e.g. all members are Elevation, not some are elevation and some are stage.)
+#. The interval and duration of each member *MAY* be different.
+
+
+Time Series Naming
+------------------
+
+Option 1
+~~~~~~~~
+
+`<Location Id>.<Parameter>.<Parameter Type>.Composite.var.<version>`
+
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+| Element              | Description                                                                                                            |
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+|Location Id           |As the normal CWMS TS ID, the location for this measure                                                                 |
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+|Parameter             |As the normal CWMS TS ID, the measurement (e.g. Stage, Precip, Elevation, flow, etc)                                    |
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+|Parameter Type        |As Normal CWMS TS ID, Instantaneous, average, total, etc                                                                |
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+|Interval -\> Composite| Marker that this time series does not have a fix information and is build of various member time series.               |
++----------------------+------------------------------------------------------------------------------------------------------------------------+
+|Duration -\> var      |Duration of average or total may change over time with new members, duration will be indicated in the member definition |
```

Comment:
I would suggest writing this out as Variable as that is already proposed in standard naming. This is of course dependent on variable durations being present given the want to have a consistent duration as noted above.